common/src/main/java/com/regnosys/rosetta/common/serialisation/mixin/RosettaJSONAnnotationIntrospector.java

```diff
@@ -125,11 +132,71 @@ public JsonIgnoreProperties.Value findPropertyIgnorals(Annotated ac) {
     private static Set<String> getPropertyNames(AnnotatedClass acc, Predicate<AnnotatedMethod> filter) {
         return StreamSupport.stream(acc.memberMethods().spliterator(), false)
                 .filter(filter)
-                .map(m -> BeanUtil.getPropertyName(m.getAnnotated()))
+                .map(m -> {
```

Comment:
The predicate passed in already filters on the `RosettaAttribute.class`. Maybe we should just have 2 methods
- `getRosettaPropertyNames`
- `getOtherPropertyNames`